code/chart.js

```diff
@@ -1,4 +1,29 @@
 //DOM-selector for the canvas 👇
-const ctx = document.getElementById('chart').getContext('2d')
+const ctx = document.getElementById("chart").getContext("2d");
 
 //"Draw" the chart here 👇
+Chart.defaults.font.size = 16;
```

Comment:
Nice! Looks more easier to change the font size here (I put it inside the datasets... 😅 )